src/Core/Domain/Product/Combination/Query/GetEditableCombinationsList.php

```diff
@@ -54,24 +54,24 @@ class GetEditableCombinationsList
     /**
      * @var int|null
      */
-    private $offset;
+    private $page;
 
     /**
      * @param int $productId
      * @param int $languageId
-     * @param int $limit
-     * @param int $offset
+     * @param int|null $limit
+     * @param int|null $page
```

Comment:
Mmm, now I get why you needed to convert the page into an offset in the handler
I think he initial thought to use offset in the command (and repository) was the right one! It offers more flexibility for more use cases, although it's a bit more technical

But I understand why it's not convenient to use in the javascript component, so I'd suggest to keep offset in command and repository, but the javascript can use a `page` parameter in the query, and it's the `CombinationController`'s responsibility to convert this `page` parameter into a proper `offset` argument for the command, seems like the right place to do this conversion

What do you think?

Reply:
so after little discussion in slack, Ill change query and controller to accept offset & limit. Page->offset conversion will be handled in paginator component. This is the way it works in current static grids, so its better to keep same behavior